.cirrus.yml

```diff
@@ -0,0 +1,18 @@
+freebsd_instance:
+  image_family: freebsd-14-0
+
+task:
+  name: "Build"
+  install_script: >
+    pkg install -y cmake ninja pkgconf
+    graphics/png graphics/webp graphics/tiff graphics/libheif graphics/libavif
+    graphics/libimagequant print/freetype2 x11/libXpm x11-fonts/fontconfig
+    print/libraqm
+  script: |
+      mkdir build
```

Comment:
please be consistent with indentation.  we use 2 spaces per-level in GH files, so might as well keep that here.